src/main/kotlin/no/nav/helse/spenn/tasks/SendTilAvstemmingTask.kt

```diff
@@ -43,6 +46,17 @@ class SendTilAvstemmingTask(val oppdragStateService: OppdragStateService,
                 return
             }
         }
+        try {
+            meldinger[1].grunnlag.apply {
+                meterRegistry.counter(AVSTEMMING, "type", "godkjent").increment(this.godkjentAntall.toDouble())
+                meterRegistry.counter(AVSTEMMING, "type", "avvist").increment(this.avvistAntall.toDouble())
+                meterRegistry.counter(AVSTEMMING, "type", "mangler").increment(this.manglerAntall.toDouble())
+                meterRegistry.counter(AVSTEMMING, "type", "varsel").increment(this.varselAntall.toDouble())
+            }
+        } catch (e : Exception) {
+            log.error("Error registering metrics", e)
```

Comment:
telle denne og?

Reply:
mnjaa... det bør egentlig ikke komme noe feil her. Det er mest for at hvis det mot formodning skulle skje noe galt ved metricene så bør ikke det hindre neste steg, som er å oppdatere avstemt-status i DB (når metrics gjøres er avstemming allerede fullført vellykket).
Tenker at error i Kibana, og det at AVSTEMMING.count vil avvike fra OPPDRAG_SENDT_TIL_OS.count kanskje er deteksjon godt nok